app.js

```diff
@@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
 
 // Merge in variable options for sitemap.
 function sitemapOptions (options) {
-    options.http = 'https';
-
     if (env === 'production') {
```

Comment:
Yeah that is what I meant :)

On Aug 12, 2016 23:39, "Joshua Mervine" <notifications@github.com> wrote:

> In app.js
> <https://github.com/MaxCDN/bootstrap-cdn/pull/737#discussion_r74656843>:
>
> > @@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
> >
> >  // Merge in variable options for sitemap.
> >  function sitemapOptions (options) {
> > -    options.http = 'https';
> > -
> >      if (env === 'production') {
>
> The check returns options unchanged for production... I could rewrite the
> whole thing as...
>
> // Merge in variable options for sitemap.function sitemapOptions (options) {
>     if (env !== 'production') {
>         options.route = {
>             '/': {
>                 disallow: true
>             }
>         };
>     }
>     return options;
> }
>
> Which is cleaner for sure.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/737/files/7c380cfac434705b88356aa66477d0ab531e41fb..9ef2f34d397eeb9e96cfb282d1306e9fb431583a#r74656843>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtcTFxza3hEtbE0toJP9A0w_bhGkxks5qfNnpgaJpZM4JeeRW>
> .
>